packages/split-button/src/SplitButton/SplitButton.tsx

```diff
@@ -26,14 +26,14 @@ import { Align, Justify, SplitButtonProps, Variant } from './SplitButton.types';
 export const SplitButton = InferredPolymorphic<SplitButtonProps, 'button'>(
   (
     {
+      children,
```

Comment:
I believe we decided on the `menuItems` API to avoid confusion on whether the children should be the menu items or the button label

Reply:
> avoid confusion on whether the children should be the menu items or the button label

That is easily expressed through a comment on the prop :) Right now the component takes `children` and just throws them away? I feel that's more confusing 🤷 and much less expressive.

Reply:
I've updated the description above with more context on the issues I see with of the current `menuItems` prop.